packages/cli/src/util/autoVersion.test.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { execSync } from "node:child_process";
+import { describe, expect, it, vi } from "vitest";
+import { getAutoVersion } from "./autoVersion.js";
+
+vi.mock("node:child_process", () => ({
```

Comment:
Have you tried removing the factory argument (just use `vi.mock("node:child_process")`)? I think vitest should automatically mock out the object with defaults